tests/test_dataset.py

```diff
@@ -420,11 +483,31 @@ def test_graph_standardization_logic(self):
             GraphDataset(
                 hdf5_path = hdf5_path,
                 target='binary',
-                standardize=True,
                 train=False,
                 dataset_train=dataset_train
             )
+            
+    def test_feature_transform_mean_std(self):
+        node_feat_test='bsa'
+        hdf5_path = "tests/data/hdf5/train.hdf5"
+        features_transform={'bsa':{'transform':lambda t:np.log(t+1),'standardize':True}}
+        
+        #calculate transformation and standardization form GraphDataset object
+        dataset_test_transform = GraphDataset(
+            hdf5_path = hdf5_path,
+            target='binary',
+            node_features = [node_feat_test],
+            edge_features =[],
+            features_transform=features_transform
+        )
+        
+        #calculate transformation and standardization maunally
+        means_devs=_cal_mean_std(hdf5_path,features_transform,node_feat_test)
```

Comment:
Following on my comments above about `_cal_mean_std`, here you'll have:
```python
mean, dev = _cal_mean_std(hdf5_path, features_transform, node_feat_test)
```

Remember to modify the asserts below as well.

Also note the spaces after punctuation like `,` to make the writing less compact and more clearer.